src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ValidateSignature.cs

```diff
@@ -27,7 +27,7 @@ public partial class JsonWebTokenHandler : TokenHandler
         /// <exception cref="SecurityTokenInvalidSignatureException">Returned by the default implementation if the token is not signed, or if the validation fails.</exception>
         /// <exception cref="SecurityTokenInvalidAlgorithmException">Returned if the algorithm is not supported by the key.</exception>
```

Comment:
nit: Will you be updating this documentation in this iteration?

Reply:
Outside of the scope of this work. 

We need to align on how we want to communicate the exceptions being "returned" as opposed to thrown, since they can be retrieved from the validation error.